src/layouts/HeaderContainer.tsx

```diff
@@ -59,6 +59,23 @@ class HeaderContainer extends React.Component<IProps, null> {
     this.props.changeAccount(newAddress);
   }
 
+  public networkName = (id: number) => {
```

Comment:
theres probably a better place for this since we could use it elsewhere. how about lib/util.ts?

Reply:
updated: https://github.com/daostack/alchemy/pull/39/commits/e8f0b5edeff4215ac060566e20b9209a1bc860b5